debezium-bom/pom.xml

```diff
@@ -22,6 +22,8 @@
         <version.netty>4.1.51.Final</version.netty>
         <!-- check new release version at https://github.com/confluentinc/schema-registry/releases -->
         <version.confluent.platform>6.0.2</version.confluent.platform>
+        <version.confluent.protobuf>6.1.0</version.confluent.protobuf>
```

Comment:
Should `version.confluent.platform` be used here from the line above? There can be big Kafka version jumps even between minor versions of confluent platform.

Reply:
Agree. I will change it.